### DISCUSSION TITLE
anyway to change the default spritecomponent rect to render with circle with customized radius?

### DISCUSSION THREAD

**Discussion by CKT90** [flame-engine/flame#2645] [General]
as per topic, can we use Rect.fromCircle() with spritecomponent to render a circular background instead of a rectangular?

thanks.

**Comment by spydon** [comment]
You can wrap your `SpriteComponent` in a `ClipComponent` to make it circular.
Or you can extend the `SpriteComponent` and override the `render` method, but the first option is more straightforward.
